### PR TITLE
Display TS compile errors + minor clean up

### PR DESCRIPTION
**Pull request by kirjs**
t3pc8adownx

### CODE REVIEW

**Comment by kirjs** [comment]
This whole file is under a BIG TODO

File: src/app/demo/typescript/typescript.component.html [hunk 1]
```diff
@@ -24,10 +24,10 @@
     <div row>
       <div>
         <ul>
-          <li><b>TypeScript</b> a language transpiled to JavaScript. It extends <b>ES5</b> standard (current
+          <li><b>TypeScript</b> a language that transpiles to JavaScript. It extends <b>ES5</b> standard (current
```
Comment (tessaSAC) [inline src/app/demo/typescript/typescript.component.html]: Will people understand transpilation?

Comment (kirjs) [inline src/app/demo/typescript/typescript.component.html]: What would be an alternative? Compilation sounds equally scary :)

**Comment by kirjs** [comment]
The change was transpiled->transpiles. 
It would be nice to have a whole pass to search for complex words and make it beginner friendly.

**Pull request merged**